docs/examples/Power Flow Example.ipynb

```diff
@@ -403,7 +402,7 @@
       "List of attribute types in line result\n",
       "['id', 'p_from']\n",
       "------line result------\n",
-      "{'id': array([3, 5, 8], dtype=int32), 'p_from': array([17360100.20222374, -3365613.74450156, 13854413.52498137])}\n"
+      "{'id': array([3, 5, 8]), 'p_from': array([17360100.20222374, -3365613.74450156, 13854413.52498137])}\n"
```

Comment:
is this change right? (either way, it's weird, because `ID` should be `int64`...)

Reply:
This is right. `ID` is always `int32`. We don't have a `int64` attribute yet.

Reply:
ohhh right, sorry, apologies. `Idx` is `int64`, `ID` is `int32`, my bad